Source Main 5.2/source/ZzzInterface.cpp

```diff
@@ -592,39 +601,46 @@ void RenderNotices()
     NoticeInverse += FPS_ANIMATION_FACTOR;
 }
 
-void CutText(const char* Text, char* Text1, char* Text2, int Length)
+void CutText(const wchar_t* Text, wchar_t* Text1, wchar_t* Text2, int Length)
```

Comment:
This method needs to be fixed... it causes some empty chat lines, with just " : "